zap/backends/postgresql.py

```diff
@@ -28,10 +28,13 @@ def _psql(self, *command):
         ''' Run a command via psql as the postgres user '''
         if self.debug:
             sys.stderr.write('psql -c "' + ' '.join(command) + '"\n')
-        base_command = ['sudo', '-u', 'postgres', 'psql', '-c']
+        port_string = '{port}'.format(port=self.port)
+        base_command = ['sudo', '-u', 'postgres',
+                        'psql', '-p', port_string, '-c']
```

Comment:
Does this definitely still work when settings.DATABASES['PORT'] is an empty string that gets passed to psql?